README.md

```diff
@@ -1,15 +1,20 @@
-**Interested in contributing a code example?** Please let us know about your
-code example by submitting a new issue to this repository. Thanks!
+**Interested in contributing a code example?** 
+
+Please take a look at our [contribution guidelines](CONTRIBUTING.md) before
+getting started. Thanks!
 
 <!-- Before submitting your code, please delete the above code contribution
 instructions and this comment as they will not be relevant in your code 
 example README.md.-->
 
 # <demo_name>
 
-Short blurb on what this demo does.
+Describe your example and specify what it is demonstrating. Does it have a
+business application? Is it pedagogical? It is meant to showcase performance? A
+clear description allows us to properly categorize your example. 
```

Comment:
Does it belong to a particular domain such as material simulation or logistics?
What level of Ocean proficiency does it target (beginner, advanced, pro)?

If you end up with more than about three question, might be better in a bullet list